libraries/linux_module.rb

```diff
@@ -48,7 +48,7 @@ def loaded?
 
   def version
     modinfo_cmd = if inspec.os.redhat? || inspec.os.name == 'fedora'
-                    "/sbin/modinfo -F version #{@module}"
+                    "modinfo -F version #{@module}"
```

Comment:
We need to be careful. The user path is only available for local execution, not necessarily for remote ssh checks. In those cases we need to switch and use the bash resource instead of the command resource